src/javascript/components/SitemapPanelApp.jsx

```diff
@@ -151,9 +158,9 @@ const SitemapPanelApp = ({client, dxContext, t}) => {
                                 <Typography className={styles.sitemapIndexFileDescription} component="p">{t('labels.settingSection.sitemapIndexFileSection.description')}</Typography>
                                 <Card>
                                     <div className={styles.sitemapIndexFileCardArea}>
-                                        <File className={(!formik.values.sitemapIndexURL || !sitemapMixinEnabled) ? styles.sitemapIndexFileIconDisable : styles.sitemapIndexFileIconEnabled} size="big"/>
-                                        <Typography className={(!formik.values.sitemapIndexURL || !sitemapMixinEnabled) ? styles.sitemapIndexFileNameDisabled : styles.sitemapIndexFileNameEnabled} component="p">{t('labels.settingSection.sitemapIndexFileSection.sitemapIndexXML')}</Typography>
-                                        <Button className={styles.sitemapIndexFileButton} variant="ghost" icon={<OpenInNew size="big"/>} disabled={!formik.values.sitemapIndexURL || !sitemapMixinEnabled} onClick={() => onOpenSitemapXMLButtonClick(formik.values.sitemapIndexURL)}/>
+                                        <File className={(formik.values.sitemapIndexURL === '' || !sitemapMixinEnabled) ? styles.sitemapIndexFileIconDisable : styles.sitemapIndexFileIconEnabled} size="big"/>
```

Comment:
should we maybe check for empty whitespace, not just empty? ie. add a trim()

also I see it used on different lines. maybe a variable for the boolean might be useful.

Reply:
It is actually a state object with **formik**
It is a bit weird with **formik** they need to specify '' as default null will cause issues for input field.
But being said it only check for empty string as this point. (For validation of the input fields we can add that logic in a different place if needed)